blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitPipelineCreateRequest.java

```diff
@@ -95,4 +98,45 @@ public BluePipeline create(Reachable parent) throws IOException {
         return null;
     }
 
+    private void validate(String name, BlueScmConfig scmConfig){
```

Comment:
Is there a way this can be exposed to the REST services so the UI could pre-validate credentials?